### ISSUE TITLE
Adamantium Plushies are OP and need to be nerfed

### ISSUE THREAD

**Issue by Smithead** [Aurorastation/Aurora.3#6079]
Here is a list of issues I've identified with plushies:
* They are not able to be picked up, which can lead to issues where one may get stuck in a maintenance shaft, and unable to get out _because a plush doll is in their way._ 
-[ref1](https://gyazo.com/4f84c1e9478fdd83ba1e8b81263a0ddb)
-[ref2](https://gyazo.com/74933191a9dd6df762ba0e241f8b36bf)
* They are essentially indestructible, and cannot be disposaled, or destroyed in the crusher.
-[ref](https://gyazo.com/8ab3655928cb7a995d671ef5395904b4)
* In short: **plushies should be able to be picked up, and thrown into disposals, where they can be _destroyed._**

GameID: bYU-cmj7
